src/fragment-components/a-radiobutton-group.tsx

```diff
@@ -0,0 +1,208 @@
+	);
+};
+
+export const componentInfo: ComponentInfo = {
+	component: ARadioButtonGroup,
+	styleUI: ARadioButtonGroupStyleUI,
+	codeUI: ARadioButtonGroupCodeUI,
+	keywords: ['radio', 'button', 'group'],
+	name: 'Radio buttons',
+	defaultComponentObj: {
+		disabled: false,
+		type: 'radioButtonGroup',
+		legend: 'Radio Button Group',
+		codeContext: {
+			formItemName: 'radio-group'
+		},
+		defaultSelected: '',
+		valueSelected: '',
+        labelPosition: 'left',
+        orientation: 'horizontal',
+		items: [
+			{
+                type: 'radioButton',
+				codeContext: {
+					formItemName: 'radio-group',
+				},
+                labelText: "Option 1",
+				disabled: false,
+				defaultChecked: true,
+			},
+			{
+                type: 'radioButton',
+				codeContext: {
+					formItemName: 'radio-group',
+				},
+                labelText: "Option 2",
+				disabled: false,
+				defaultChecked: false,
+				
+			},
+			{
+                type: 'radioButton',
+				codeContext: {
+					formItemName: 'radio-group'
+				},
+                labelText: "Option 3",
+				disabled: false,
+				defaultChecked: false,
+			}
+		]
+	},
+	render: ({ componentObj, select, remove, selected, renderComponents }) => <ARadioButtonGroup
+		componentObj={componentObj}
+		select={select}
+		remove={remove}
+		selected={selected}>
+		{componentObj.items.map((button: any) => (renderComponents(button)))}
+	</ARadioButtonGroup>,
+	image,
+	codeExport: {
+		angular: {
+			inputs: ({ json }) => `@Input() ${nameStringToVariableString(json.codeContext?.name)}LegendText = "${json.legend}";
+								@Input() ${nameStringToVariableString(json.codeContext?.name)}Orientation = "${json.orientation}";
+								@Input() ${nameStringToVariableString(json.codeContext?.name)}LabelPosition = "${json.labelPosition}";
+								@Input() ${nameStringToVariableString(json.codeContext?.name)}Name = "${json.codeContext?.formItemName}";
+								@Input() ${nameStringToVariableString(json.codeContext?.name)}defaultSelected = "${json.defaultSelected}";
+								@Input() ${nameStringToVariableString(json.codeContext?.name)}valueSelected = "${json.valueSelected}";`,
+			outputs: ({ json }) => ``,
+			imports: ['RadioModule'],
+			code: ({ json, jsonToTemplate }) => {
+				return `
+				<legend class="bx--label">{{${nameStringToVariableString(json.codeContext?.name)}LegendText}}</legend>
+				<ibm-radio-group
+					[name]="${nameStringToVariableString(json.codeContext?.name)}Name"
+					[orientation]="${nameStringToVariableString(json.codeContext?.name)}Orientation"
+					[labelPlacement]="${nameStringToVariableString(json.codeContext?.name)}LabelPosition"
+					${angularClassNamesFromComponentObj(json)}>
+						${json.items.map((element: any) => jsonToTemplate(element)).join('\n')}
+				</ibm-radio-group>`
+			}
+		},
+		react: {
+			imports: ['RadioButtonGroup'],
+			code: ({ json, fragments, jsonToTemplate }) => {
+				return `
+				<RadioButtonGroup
+					name="${json.codeContext?.formItemName}"
```

Comment:
What if the user does not set the formItemName? I think it is safer to use the name here - Refer to updated RadioTiles